roles/openshift_logging/defaults/main.yml

```diff
@@ -13,10 +13,26 @@ openshift_logging_curator_run_minute: 0
 openshift_logging_curator_run_timezone: UTC
 openshift_logging_curator_script_log_level: INFO
 openshift_logging_curator_log_level: ERROR
+openshift_logging_curator_cpu_limit: 100m
+openshift_logging_curator_memory_limit: null
```

Comment:
I don't think exactly.  
- If you do not provide them at all then you get an error in evaluating the task saying the vars are undefined.  This happens before getting to the jinja template
- If you add in the default filter and set them to something like 'default(None)' then in the jinja template it gets evaluated to the string value 'None'.  This makes the if block undesirable IMO.  You should be able to do: 'if VAR is not none'.  'null' is the only way i found for jinja to recognize the vale as 'none'

An alternate point that I do like is defining it explicitly shows someone what is possible without requiring them to dig into the template or tasks.